src/test/scala/cognite/spark/v1/RawTableRelationTest.scala

```diff
@@ -659,19 +659,19 @@ class RawTableRelationTest
 
     assert(df1.count() == 2)
 
-    val rowsRead1 = getNumberOfRowsRead(metricsPrefix1, f"raw.spark-test-database.${tableName}.rows")
+    val rowsRead1 = getNumberOfRowsRead(metricsPrefix1, s"raw.spark-test-database.$tableName.rows")
     assert(rowsRead1 == 2)
 
     // filter should not be pushed down if OR condition includes other fields
     val metricsPrefix2 = s"pushdown.raw.key.${shortRandomString()}"
     val df2 = rawRead(tableName, metricsPrefix = Some(metricsPrefix2))
       .where(
-        "key = 'some-invalid-key' or key = 'k1' or key = 'k2' or lastUpdatedTime >= timestamp('2000-01-01 00:00:00.000Z')")
+        "key = 'some-invalid-key' or key = 'k1' or key = 'k2' or key = 'k1' or lastUpdatedTime >= timestamp('2000-01-01 00:00:00.000Z')")
 
-    assert(df2.count() == 3)
+    assert(df2.count() == 2)
 
-    val rowsRead2 = getNumberOfRowsRead(metricsPrefix2, f"raw.spark-test-database.${tableName}.rows")
-    assert(rowsRead2 == 3)
+    val rowsRead2 = getNumberOfRowsRead(metricsPrefix2, s"raw.spark-test-database.$tableName.rows")
+    assert(rowsRead2 == 2)
```

Comment:
This query should return all 3 rows, since all have `lastUpdatedTime >= timestamp('2000-01-01 00:00:00.000Z')`
(see above comment)

```suggestion
    assert(df2.count() == 3)

    val rowsRead2 = getNumberOfRowsRead(metricsPrefix2, s"raw.spark-test-database.$tableName.rows")
    assert(rowsRead2 == 3)
```